resources/schemas/dbscripts/postgresql/targetedms-21.002-21.003.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN TraceValue REAL;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN TimeValue REAL;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN Trace BIGINT;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN YAxisLabel VARCHAR(200);
```

Comment:
Can we use the existing Series1Label column instead of introducing a separate one just for traces?